composer.json

```diff
@@ -14,13 +14,20 @@
 			"email": "jeroendedauw@gmail.com",
 			"homepage": "http://jeroendedauw.com",
 			"role": "Developer"
+		},
+		{
+			"name": "Daniel Kinzler"
 		}
 	],
 	"support": {
 		"irc": "irc://irc.freenode.net/wikidata"
 	},
 	"require": {
-		"php": ">=5.3.0"
+		"php": ">=5.3.0",
+		"wikimedia/assert": "0.2.2"
+	},
+	"require-dev": {
+		"phpunit/phpunit": "3.7.37"
```

Comment:
How is the version of PHPUnit that MediaWiki uses relevant to a PHP library that does not depend on MediaWiki?!

And no, you don't need this to be able to run the tests, as evidenced by the tests passing on master before this change